Cubical/Experiments/ZariskiLatticeBasicOpens.agda

```diff
@@ -1,249 +0,0 @@
-{-# OPTIONS --safe --lossy-unification #-}
```

Comment:
@mzeuner    Should this be kept or deleted?

Reply:
The affine schemes paper mentions this file as a "A first, now deprecated, formalization attempt" in the conclusion.
So, maybe keep it? That is, if we're keeping some of those files anyways. I don't have a strong opinion on that.
I'm pretty sure it doesn't type check anymore, but it should be easy to fix that.

Reply:
This one does not check.

Reply:
Partially fixed, commented out rest, added to Everything